rundeckapp/grails-app/services/rundeck/services/LogFileStorageService.groovy

```diff
@@ -1369,6 +1373,60 @@ class LogFileStorageService
         return null
     }
 
+    /**
+     * @param projectName of the desired project to get the storage path for logs
+     * @return the value of the framework property "framework.plugin.ExecutionFileStorage.<file-storage-plugin>.path"
+     */
+    String getStorePathForProject(String projectName){
+        PropertyResolverFactory.Factory resolverFactory = frameworkService.getFrameworkPropertyResolverFactory(projectName)
+        String pluginName = getConfiguredPluginName()
+        String storePath = null
+
+        if(pluginName != null) {
+            Description description = pluginService.getPluginDescriptor(pluginName, executionFileStoragePluginProviderService).getDescription()
+            storePath = PluginAdapterUtility.mapDescribedProperties(resolverFactory.create(executionFileStoragePluginProviderService.name, description), description, PropertyScope.Instance).get('path')
```

Comment:
it is not right to rely on a plugin property (right now might be path but it could change and could have any name for any implementation), instead it should be part of the exposed interfaces